src/components/Quote.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Calculator from './Calculator';
+
+const Quote = ({ category }) => {
+  const [quote, setQuote] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    // make the HTTP request to the API
+    const url = `https://api.api-ninjas.com/v1/quotes?category=${category}`;
+    fetch(url)
+      .then((response) => response.json())
+      .then((data) => {
+        // update the state with the first quote
+        setQuote(data[0].quote);
+        setLoading(true);
+        setError(true);
+      })
+      .catch(() => {
+        setLoading(true);
+        setError(false);
+      });
+  }, [category]);
```

Comment:
- [ ] Please check your code your API is not working I think you missed `API KEY` that's why it's not working so kindly add `API KEY` and check your end it's working or not also check below :point_down: the error message :slightly_smiling_face: 

![Screenshot from 2023-06-02 15-41-42](https://github.com/theresetuyi/Math-magician/assets/45226848/3d6da05e-ec5a-46ed-bafe-e2c0f4acc323)